apps/ios/Shared/Views/UserSettings/UserProfilesView.swift

```diff
@@ -37,8 +38,18 @@ struct UserProfilesView: View {
         .toolbar { EditButton() }
     }
 
-    private func userProfileView(_ userBinding: Binding<UserInfo>, index: Int) -> some View {
-        let user = userBinding.wrappedValue.user
+    private func removeUser(index: Int) {
+        do {
+            try apiDeleteUser(m.users[index].user.userId)
+            var users = m.users
+            users.remove(at: index)
+            m.updateUsers(users)
+        } catch {
+            fatalError("Failed to delete user: \(responseError(error))")
```

Comment:
alert instead of fatalError?

Reply:
yes, in another PR as deletion needs to be changed to swipe actions anyway